tests/e2e/cucumber/features/admin-settings/spaces.feature

```diff
@@ -105,11 +105,11 @@ Feature: spaces management
       | name   | id     |
       | team A | team.a |
     And "Admin" adds the following members to the space "team A" using API
-      | user  | role     | shareType |
-      | Brian | Can edit | space     |
-      | Carol | Can view | space     |
-      | David | Can view | space     |
-      | Edith | Can view | space     |
+      | user  | role     | resourceType |
```

Comment:
Please, also check if we need this
```suggestion
      | user  | role     |
```

Reply:
yes, we need this for roleId

Reply:
hmm, is it possible to not use this in a feature file but directly use it in the functions wherever needed. because the step is to add user to space. So I don't see why we need it. but I might be wrong here.

Reply:
If we add user, then his role can be for instance `can view`. But it is converted as `viewer`. But we need to specify if the viewer is either: `viewer` or `space viewer` or `secure viewer` .
Here, resourceType indicates either the role is normal or space or secure.
Similarly, in case of editor, we should specify, if it is file editor, space editor or or normal editor.

Reply:
Ah, I am talking about this data:
```
| resourceType |
| space        |
```
sorry if you got it wrong :sweat_smile:

Reply:
yes, i'm also talking about the same 😅
if we pass resourceType as space, then only we can filter the role id as `space editor` else only `editor` will be passed. 
I figured out this way. 
I welcome any better suggestions to get the apropriate roleId